packages/field-views-loader/index.js

```diff
@@ -62,60 +86,66 @@ module.exports = function() {
 
   let allPaths = new Set();
 
-  const stringifiedObject = serialize(
-    Object.entries(adminMeta.lists).reduce((obj, [listPath, { views }]) => {
+  let pageComponents = findPageComponents(adminMeta.pages);
+
+  let allViews = Object.entries(adminMeta.lists).reduce(
+    (obj, [listPath, { views }]) => {
       obj[listPath] = views;
       return obj;
-    }, {}),
-    allPaths
+    },
+    { __pages__: pageComponents }
   );
 
+  const stringifiedObject = serialize(allViews, allPaths);
+
   let loaders = `{\n${[...allPaths]
     .map(path => {
       return `'${path}': () => import('${path}').then(interopDefault)`;
     })
     .join(',\n')}\n}`;
 
-  return `
-  let promiseCache = new Map();
-  let valueCache = new Map();
+  const source = endent`
+    let promiseCache = new Map();
+    let valueCache = new Map();
```

Comment:
Not a thing about this PR but we should probably move all this suspense stuff into the Admin UI since it doesn't really need to be in the loader and we can have prettier, eslint and etc. on it.